File: app/src/main/java/jp/superwooo/chaipon/lighttimeswitcher/LimitTime.java
package jp.superwooo.chaipon.lighttimeswitcher;

public class LimitTime {
    private final int _min;
    private final int _max;
    public LimitTime(int min, int max){
        _min = min;
        _max = max;
    }
    public int apply(int value){
        return Math.max(_min, Math.min(_max, value));
    }
}

File: app/src/main/java/jp/superwooo/chaipon/lighttimeswitcher/LimitTime.kt
package jp.superwooo.chaipon.lighttimeswitcher

import kotlin.math.max
import kotlin.math.min

class LimitTime(private val _min: Int, private val _max: Int) {
    fun apply(value: Int): Int {
        return max(_min.toDouble(), min(_max.toDouble(), value.toDouble())).toInt()
    }
}
